rotkehlchen/data_import/importers/binance.py

```diff
@@ -40,6 +45,7 @@
 
 BinanceCsvRow = dict[str, Any]
 BINANCE_TRADE_OPERATIONS = {'Buy', 'Sell', 'Fee'}
+EVENT_IDENTIFIER = f'{ROTKI_EVENT_PREFIX}BNC'
```

Comment:
Just use BNC. Shorter and not used, right? No need to add the RE prefix. Or is there something I am missing?